### PR TITLE
Fixes informational suggestions in Sherlock 9 and 10.

### PR DESCRIPTION
**Pull request by MikeHathaway**
Fixes informational suggestions in Sherlock 9 and 10.

- Create new Constants and utility functions to remove magic numbers like `screeningStageEndBlock`
- Update comments
- Cache array length to reduce gas cost of loops
- Update an extraordinary funding revert simplification suggested by dmitri: https://github.com/ajna-finance/ecosystem-coordination/pull/39#pullrequestreview-1294783038

### CODE REVIEW

File: src/grants/base/StandardFunding.sol [hunk 1]
```diff
@@ -27,11 +27,24 @@ abstract contract StandardFunding is Funding, IStandardFunding {
...
+     * @dev    Roughly equivalent to the number of blocks in 90 days.
      */
-    uint256 internal constant DISTRIBUTION_PERIOD_LENGTH = 648000; // 90 days
+    uint256 internal constant DISTRIBUTION_PERIOD_LENGTH = 648000;
+
+    /**
+     * @notice Length of the funding phase of the distribution period in blocks.
+     * @dev    Roughly equivalent to the number of blocks in 10 days.
+     */
+    uint256 internal constant FUNDING_PERIOD_LENGTH = 72000;
```
Comment (EdNoepel) [inline src/grants/base/StandardFunding.sol:47]: Confirmed all the numbers above are correct for 12 second blocks.

**Review by EdNoepel — APPROVED** [review]
LGTM

**Review by ith-harvey — APPROVED** [review]
LGTM

**Review by dmitriia — COMMENTED** [review]
Looks ok

**Pull request merged**